cypress.config.ts

```diff
@@ -12,6 +12,7 @@ module.exports = defineConfig({
   responseTimeout: 60000,
   viewportWidth: 2000,
   viewportHeight: 1320,
+  retries: 2,
```

Comment:
I'll remove this. This was in response to Federico's PR which can have different results with retries, but since he added it in his PR, there is no need to have this here.